From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sun, 6 Feb 2022 22:23:00 +0100
Subject: [PATCH] Use ash (vulkan) in linked mode

This depends on cargo.patch - enabling feature "linked" for ash crate.

--- a/src/frame/processor/vulkan.rs
+++ b/src/frame/processor/vulkan.rs
@@ -100,7 +100,7 @@
             vk::KhrGetPhysicalDeviceProperties2Fn::name().as_ptr(),
         ];
 
-        let entry = unsafe { Entry::load()? };
+        let entry = Entry::linked();  // XXX-Patched
 
         let create_info = vk::InstanceCreateInfo::builder()
             .application_info(&app_info)
